### PR TITLE
chore: simplify CreateDemand implementation

### PR DESCRIPTION
**Pull request by soanvig**
Either way it's breaking change, so we must decide whether to include end date or not (the default value on backend)

### CODE REVIEW

File: packages/ui/exchange-ui-core/src/components/modal/DemandUpdateModal.tsx [hunk 1]
```diff
@@ -102,8 +102,7 @@ export function DemandUpdateModal(props: IProps) {
                 start: moment(values.start).toISOString(),
                 end: moment(values.end).toISOString(),
                 product: values.product,
-                boundToGenerationTime: false,
-                excludeEnd: false
```
Comment (soanvig) [inline packages/ui/exchange-ui-core/src/components/modal/DemandUpdateModal.tsx:106]: I set default excludeEnd to `true` on backend

File: packages/ui/exchange-ui-core/src/pages/ViewMarketPage/hooks/useBidHandler.ts [hunk 2]
```diff
@@ -68,8 +68,7 @@ export async function bidHandler({
                         ? undefined
                         : values.location?.map((l) => `${country};${l}`)
                 },
-                boundToGenerationTime: false,
-                excludeEnd: false
```
Comment (soanvig) [inline packages/ui/exchange-ui-core/src/pages/ViewMarketPage/hooks/useBidHandler.ts:72]: I set default excludeEnd to `true` on backend

File: packages/ui/exchange-ui-core/src/utils/exchange/math.ts [hunk 3]
```diff
@@ -23,15 +23,13 @@ export async function calculateTotalVolume(
...
-                parsedVolume
-            ).toString(),
+            volumePerPeriod:
+                EnergyFormatter.getBaseValueFromValueInDisplayUnit(parsedVolume).toString(),
             periodTimeFrame: period,
             start: start.toISOString(),
             end: end.toISOString(),
             product: {},
-            boundToGenerationTime: false,
-            excludeEnd: false
```
Comment (soanvig) [inline packages/ui/exchange-ui-core/src/utils/exchange/math.ts:34]: I set default excludeEnd to `true` on backend

**Comment by kosecki123** [comment]
What's the reason for this change and what does this simplify? It was actually explicitly added here https://github.com/energywebfoundation/origin/pull/1342

**Comment by soanvig** [comment]
> What's the reason for this change and what does this simplify? It was actually explicitly added here #1342

I know, let's wait for @JosephBagaric with specific explanation, because I'm not entirely sure. I just got info what should be done :)

**Review by kosecki123 — APPROVED** [review]

**Pull request merged**